groups/voteForDelegate.js

```diff
@@ -0,0 +1,67 @@
+const axios = require('axios');
+const logger = require('../helpers/logger');
+const keys = require('../helpers/keys');
+const constants = require('../helpers/constants');
+const transactionFormer = require('../helpers/transactionFormer');
+const validator = require('../helpers/validator');
+
+const DEFAULT_SEND_TOKENS_RETRIES = 4; // How much re-tries for send tokens requests by default. Total 4+1 tries
+
+module.exports = (nodeManager) => {
+  /**
+		* Creates votes for delegate transaction, signs it, and broadcasts to ADAMANT network
+    * See https://github.com/Adamant-im/adamant/wiki/Transaction-Types#type-3-vote-for-delegate-transaction
+    * @param {string} passPhrase Senders's passPhrase. Sender's address will be derived from it.
+    * @param {string[]} votes PublicKeys for upvote and downvote
+    * @param {number} maxRetries How much times to retry request
+    * @returns {Promise} Request results
+  	*/
+	return (passPhrase, votes, maxRetries = DEFAULT_SEND_TOKENS_RETRIES, retryNo = 0) => {
```

Comment:
`api.voteForDelegate(config.passPhrase, ['c0c580c3fb89409f32181fef58935f286f0c1bbf61bd727084ed915b3a4bc95b'])` failed with `Wrong 'votes' parameter` error

Reply:
My fault, it should be '+c0c580c3fb89409f32181fef58935f286f0c1bbf61bd727084ed915b3a4bc95b'